stable/pomerium/values.yaml

```diff
@@ -0,0 +1,133 @@
+  rootDomain: corp.pomerium.io
+  # existingSecret:
+  # existingConfig:
+  sharedSecret: ""
+  cookieSecret: ""
+  generateTLS: true
+  policyFile: |-
+    - from: httpbin.corp.pomerium.io
+      to: http://httpbin
+      allowed_domains:
+      - pomerium.io
+    - from: external-httpbin.corp.pomerium.io
+      to: httpbin.org
+      allowed_domains:
+      - gmail.com
+    - from: weirdlyssl.corp.pomerium.io
+      to: http://neverssl.com
+      allowed_users:
+      - bdd@pomerium.io
+      allowed_groups:
+      - admins
+      - developers
+    - from: hello.corp.pomerium.io
+      to: http://hello:8080
+      allowed_groups:
+      - admins
+
+authenticate:
+  name: authenticate
+  redirectUrl: ""
+  # see https://www.pomerium.io/docs/identity-providers.html
+  idp:
+    provider: google
+    clientID: "REPLACE_ME"
+    clientSecret: "REPLACE_ME"
+    url: ""
+    serviceAccount: ""
+  tls:
+    cert: ""
+    key: ""
+    defaultSANList: []
+    defaultIPList: []
+
+authorize:
+  name: authorize
+  tls:
+    cert: ""
+    key: ""
+    defaultSANList: []
+    defaultIPList: []
+
+proxy:
+  name: proxy
+  tls:
+    cert: ""
+    key: ""
+    defaultSANList: []
+    defaultIPList: []
+  authenticateServiceUrl: ""
+  authorizeServiceUrl: ""
+  authorizeInternalUrl: ""
+  authenticateInternalUrl: ""
+
+service:
+  # Service type can be set to ClusterIP, NodePort or LoadBalancer.
+  type: ClusterIP
+  externalPort: 443
+  annotations:
+    {}
+    # ===  GKE load balancer tweaks
+    # cloud.google.com/app-protocols: '{"https":"HTTPS"}'
+    # kubernetes.io/ingress.allow-http: "false"
+  labels: {}
+
+ingress:
+  secretName: ""
+  secret:
+    name: ""
+    cert: ""
+    key: ""
+  enabled: true
+  annotations:
+    {}
+    # === nginx tweaks
+    # kubernetes.io/ingress.class: nginx
+    # nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+    # nginx.ingress.kubernetes.io/proxy-buffer-size: "16k"
+    # ===  GKE load balancer tweaks ==
+    # cloud.google.com/app-protocols: '{"https":"HTTPS"}'
+    # kubernetes.io/ingress.allow-http: "false"
+resources:
+  {}
+  # limits:
+  #   cpu: 100m
+  #   memory: 300Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 300Mi
+
+priorityClassName: ""
+
+# Affinity for pod assignment
+# Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+# affinity: {}
+
+# Tolerations for pod assignment
+# Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+# tolerations: []
+
+# Node labels for pod assignment
+# Ref: https://kubernetes.io/docs/user-guide/node-selection/
+# nodeSelector: {}
+
+podAnnotations: {}
+podLabels: {}
+replicaCount: 1
+
+# For any other settings that are optional. for a complete listing see:
+# https://www.pomerium.io/docs/config-reference.html
+extraEnv: {}
+extraArgs: {}
+extraVolumes: {}
+
+image:
+  repository: "pomerium/pomerium"
+  tag: "v0.0.3"
+  pullPolicy: "Always"
```

Comment:
When pinned to a tag the `pullPolicy` can be `ifNotPresent` so that caching can be used locally. Can you please update.

Reply:
Yes, will do.